To-DoList/app/src/main/java/com/ezequielc/to_dolist/TasksActivity.java

```diff
@@ -0,0 +1,61 @@
+package com.ezequielc.to_dolist;
+
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+
+public class TasksActivity extends AppCompatActivity {
+    private EditText mTaskEditText, mDescriptionEditText;
+    private Button mAddButton;
+    private RecyclerView mRecyclerView;
+    private TasksAdapter mTasksAdapter;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_tasks);
+
+        int selectedDay = getIntent().getIntExtra("selectedDay", -1);
+        if (selectedDay == -1){
+            finish();
```

Comment:
Display some sort of error message to the user before you send them back to the previous screen, otherwise they will have no idea what happened.